src/game/area.c

```diff
@@ -410,6 +424,10 @@ void render_game(void) {
         } else
             gDPSetScissor(gDisplayListHead++, G_SC_NON_INTERLACE, 0, gBorderHeight, SCREEN_WIDTH,
                           SCREEN_HEIGHT - gBorderHeight);
+        
+        #ifdef DEBUG_F3DEX3_PROFILER
+            draw_f3dex3_profiler();
+        #endif
```

Comment:
Should we move this to outside the if/else block so that it always renders?

Also nitpick: Unindent preprocessor stuff here to syntactically match the rest of the function/game